### ISSUE TITLE
Doesn't work on Ubuntu 20.04 with Wacom CTL-490

### ISSUE THREAD

**Issue by ardianumam** [linuxwacom/input-wacom#306]
Hi,
I follow [these steps](https://github.com/linuxwacom/input-wacom/wiki/Installing-input-wacom-from-source) and try several times but it still doesn't work. I use Wacom CTL-490 on Ubuntu 20.04, 64bit. 
Here is the output:
1. Running this: `if test -x ./autogen.sh; then ./autogen.sh; else ./configure; fi && make && sudo make install || echo "Build Failed"`
Output:
```
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
checking for gawk... no
checking for mawk... mawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether to enable maintainer-specific portions of Makefiles... no
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking whether make supports the include directive... yes (GNU style)
checking dependency style of gcc... none
checking if build path '/home/ardian/software/input-wacom-0.48.0' has spaces... no
checking for kernel type... Linux
checking for linux-based kernel... yes
checking for kernel source/headers... /lib/modules/5.13.0-28-generic/build
checking kernel version... 5.13.0-28-generic
checking RHEL7 minor release... 0
checking power supply version... rm: cannot remove '.autoconf/build/conftest.o': Permission denied
rm: cannot remove '.autoconf/build/modules.order': Permission denied
rm: cannot remove '.autoconf/build/.modules.order.cmd': Permission denied
rm: cannot remove '.autoconf/build/conftest.mod': Permission denied
rm: cannot remove '.autoconf/build/Makefile': Permission denied
rm: cannot remove '.autoconf/build/conftest.c': Permission denied
rm: cannot remove '.autoconf/build/.conftest.o.cmd': Permission denied
rm: cannot remove '.autoconf/build/.conftest.mod.cmd': Permission denied
rm: cannot remove '.autoconf/build/conftest.mod.c': Permission denied
./configure: line 3678: .autoconf/build/Makefile: Permission denied
pre-v4.1
checking CONFIG_MODULE_SIG... y
checking CONFIG_MODULE_SIG_FORCE... 
checking CONFIG_LOCK_DOWN_KERNEL... 
checking CONFIG_LOCK_DOWN_IN_EFI_SECURE_BOOT... 
checking secure boot state... on
checking if modules must be signed... recommended
checking CONFIG_MODULE_SIG_HASH... sha512
checking for module signing hash algorithm... sha512
checking for kernel autogenerated key at /lib/modules/5.13.0-28-generic/build/certs/signing_key.pem... no
checking for kernel autogenerated cert at /lib/modules/5.13.0-28-generic/build/certs/signing_key.x509... no
checking for shim MOK key at /var/lib/shim-signed/mok/MOK.priv... no
checking for shim MOK cert at /var/lib/shim-signed/mok/MOK.der... no
checking for rEFInd MOK key at /etc/refind.d/keys/refind_local.key... no
checking for rEFInd MOK cert at /etc/refind.d/keys/refind_local.cer... no
checking for module signing key... 
checking for module signing certificate... 
configure: WARNING: Kernel module signing is recommended but we are unable to find signing keys. Consider re-running configure with the options '--with-signing-key=<key>' and '--with-signing-cert=<cert>' set.

checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating 3.7/Makefile
config.status: creating 3.17/Makefile
config.status: creating 4.5/Makefile
config.status: creating config.h
config.status: config.h is unchanged
config.status: executing depfiles commands
configure:
----------------------------------------
  BUILD ENVIRONMENT:
       linux kernel - yes 4.5
      kernel source - yes /lib/modules/5.13.0-28-generic/build

NOTE: The kernel drivers included in this package are only tested with the
X Wacom driver built from xf86-input-wacom. If you are running an X server
version older than 1.7, please use the drivers provided by the linuxwacom
package.

Please run 'make && make install'.
make  all-recursive
make[1]: Entering directory '/home/ardian/software/input-wacom-0.48.0'
Making all in 4.5
make[2]: Entering directory '/home/ardian/software/input-wacom-0.48.0/4.5'
    Building input-wacom drivers for 4.5 kernel.
make -C /lib/modules/5.13.0-28-generic/build M=/home/ardian/software/input-wacom-0.48.0/4.5
make[3]: Entering directory '/usr/src/linux-headers-5.13.0-28-generic'
make[3]: Leaving directory '/usr/src/linux-headers-5.13.0-28-generic'
make[2]: Leaving directory '/home/ardian/software/input-wacom-0.48.0/4.5'
make[2]: Entering directory '/home/ardian/software/input-wacom-0.48.0'
make[2]: Leaving directory '/home/ardian/software/input-wacom-0.48.0'
make[1]: Leaving directory '/home/ardian/software/input-wacom-0.48.0'
Making install in 4.5
make[1]: Entering directory '/home/ardian/software/input-wacom-0.48.0/4.5'
make -C /lib/modules/5.13.0-28-generic/build M=/home/ardian/software/input-wacom-0.48.0/4.5 modules_install mod_sign_cmd=''
make[2]: Entering directory '/usr/src/linux-headers-5.13.0-28-generic'
arch/x86/Makefile:148: CONFIG_X86_X32 enabled but no binutils support
  INSTALL /lib/modules/5.13.0-28-generic/extra/wacom.ko
  SIGN    /lib/modules/5.13.0-28-generic/extra/wacom.ko
At main.c:160:
- SSL error:02001002:system library:fopen:No such file or directory: ../crypto/bio/bss_file.c:69
- SSL error:2006D080:BIO routines:BIO_new_file:no such file: ../crypto/bio/bss_file.c:76
sign-file: certs/signing_key.pem: No such file or directory
  INSTALL /lib/modules/5.13.0-28-generic/extra/wacom_w8001.ko
  SIGN    /lib/modules/5.13.0-28-generic/extra/wacom_w8001.ko
At main.c:160:
- SSL error:02001002:system library:fopen:No such file or directory: ../crypto/bio/bss_file.c:69
- SSL error:2006D080:BIO routines:BIO_new_file:no such file: ../crypto/bio/bss_file.c:76
sign-file: certs/signing_key.pem: No such file or directory
  DEPMOD  /lib/modules/5.13.0-28-generic
Warning: modules_install: missing 'System.map' file. Skipping depmod.
make[2]: Leaving directory '/usr/src/linux-headers-5.13.0-28-generic'
mkdir -p /etc/depmod.d
echo "override wacom * extra" > /etc/depmod.d/input-wacom.conf
echo "override wacom_w8001 * extra" >> /etc/depmod.d/input-wacom.conf
PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin:/bin:/sbin" depmod -a 5.13.0-28-generic
/usr/sbin/update-initramfs -u -k 5.13.0-28-generic
update-initramfs: Generating /boot/initrd.img-5.13.0-28-generic
I: The initramfs will attempt to resume from /dev/nvme0n1p1
I: (UUID=08b9ffa5-bb71-4fe1-aa68-8aa1ba750fcb)
I: Set the RESUME variable to override this.
make[1]: Leaving directory '/home/ardian/software/input-wacom-0.48.0/4.5'
make[1]: Entering directory '/home/ardian/software/input-wacom-0.48.0'
make[2]: Entering directory '/home/ardian/software/input-wacom-0.48.0'
make[2]: Nothing to be done for 'install-exec-am'.
make[2]: Nothing to be done for 'install-data-am'.
make[2]: Leaving directory '/home/ardian/software/input-wacom-0.48.0'
make[1]: Leaving directory '/home/ardian/software/input-wacom-0.48.0'
```

...Then I restart my PC...

2. Running this `grep "" /sys/module/wacom*/version` with my wacom connected to my PC, output:
` grep: /sys/module/wacom*/version: No such file or directory`

3. Running this `modinfo wacom | grep version` and the output: 
```
version:        v2.00-0.48.0
srcversion:     9069AD5A48C1B017DE804CA
vermagic:       5.13.0-28-generic SMP mod_unload modversions 
ardian@ardian:~/software/input-wacom-0.48.0$ sudo rmmod wacom
```
4. Running this `sudo rmmod wacom` and the output: `rmmod: ERROR: Module wacom is not currently loaded`

Any direction what I should do? I already try ver. `0.45.0` to `0.48.0` and give me same result: still not working.
Many thanks!

**Comment by jigpu** [comment]
Its possible that this issue is caused by the kernel refusing to load an unsigned kernel module. The build system is saying that signing is "recommended", which means that it can't definitively determine if the kernel will work properly or not with unsigned modules. (See https://github.com/linuxwacom/input-wacom/wiki/Kernel-Module-Signing for more info)

Please reboot with the tablet connected, run `dmesg > dmesg.txt` and attach the dmesg.txt file to this issue. It may have some indication of why the module is not loading.

**Comment by Pinglinux** [comment]
It's been more than 3 months since Jason made his follow-up request. No reply yet. The issue was probably fixed in one of the 20.04 updates.
